lib/susedistribution.pm

```diff
@@ -464,8 +464,12 @@ sub ensure_user {
     type_string("su - $user\n") if $user ne 'root';
 }
 
-# callback whenever a console is selected for the first time
-# accepts agruments provided to select_console
+# Callback whenever a console is selected for the first time. accepts agruments
```

Comment:
I guess we should use uppercase after dot (sorry for nitpicking).

Reply:
No problem at all. Thanks.